baggage/baggage.go

```diff
@@ -21,8 +21,8 @@ import (
 	"go.opentelemetry.io/otel/label"
 )
 
-// Baggage returns a copy of the baggage in ctx.
-func Baggage(ctx context.Context) label.Set {
+// Set returns a copy of the baggage in ctx.
```

Comment:
It would probably help if we use the word set in context to help explain what it means here.

```suggestion
// Set returns a copy of the set of baggage key-values in ctx.
```